spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -1212,6 +1212,157 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("correlation") {
+    withSQLConf(CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true") {
+      Seq(false).foreach { cometColumnShuffleEnabled =>
```

Comment:
Is it intentional that we only test for `cometColumnShuffleEnabled==false`? I have similar questions for `parquet.enable.dictionary` and `nullOnDivideByZero`, where we also iterate over a single boolean and don't test both for true and false.

Reply:
Should be both `true` and `false`. I removed one of them so it's easier for me to debug. Forgot to put back.